docs/src/runtime/programs.md

```diff
@@ -27,15 +27,15 @@ transfer lamports from System Program owned accounts and pay transaction fees.
 
 ## Config Program
 
-Add configuration data to the chain and the list of public keys that are permitted to modify it
+Add configuration data to the chain, alongside the list of public keys that are allowed to modify it
 
 - Program id: `Config1111111111111111111111111111111111111`
 - Instructions: [config_instruction](https://docs.rs/solana-config-program/VERSION_FOR_DOCS_RS/solana_config_program/config_instruction/index.html)
 
 Unlike the other programs, the Config program does not define any individual
-instructions. It has just one implicit instruction, a "store" instruction. Its
-instruction data is a set of keys that gate access to the account, and the
-data to store in it.
+instructions. It has just one implicit instruction: "store". Its
+instruction data is a set of keys that gate access to the account and the
+data stored inside it.
```

Comment:
```suggestion
data stored inside of it.
```

Reply:
The instruction data is a set of keys and the data to store in the account. The change makes it seem like the instruction data is the data already stored in the account, which is incorrect. Maybe "the data to store inside of it" is clearest?